### PR TITLE
Reduce Firebase reads

### PR DESCRIPTION
**Pull request by ianmah**

### CODE REVIEW

File: src/containers/Announcements.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default () => {
       .doc(DB_HACKATHON)
       .collection('Announcements')
       .orderBy('timestamp', 'desc')
+      .limit(6)
       .onSnapshot(querySnapshot => {
```
Comment (jackyzha0) [inline src/containers/Announcements.js:16]: can we reuse the data fetched from `src/App.js:L117`?

File: src/containers/Quicklinks.js [hunk 2]
```diff
@@ -17,11 +17,13 @@ const ButtonContainer = styled.div`
...
   }
 `
-const getLinks = () => {
+
+const getCommonLinks = () => {
   return db
     .collection(DB_COLLECTION)
     .doc(DB_HACKATHON)
     .collection('QuickLinks')
+    .where('common', '==', true)
```
Comment (jackyzha0) [inline src/containers/Quicklinks.js:26]: nice

File: src/containers/Announcements.js (hunk 1 above)
Comment (ianmah) [reply src/containers/Announcements.js]: let's get someone with iOS to test the preview pr to make sure my changes didn't break anything before we merge. @acchiang?

Comment (jackyzha0) [reply src/containers/Announcements.js]: just got carmen to test on iOS, looks good

**Review by jackyzha0 — APPROVED** [review]

**Pull request merged**